content/blog/microcks-realm-fix-for-apicurio.md

```diff
@@ -0,0 +1,154 @@
+
+```
+git clone https://github.com/Apicurio/apicurio-studio.git
+```
+
+- Cd to the directory to enter the APICurio docker-compose installation workspace. Its now time to make a few edits
+
+```
+cd apicurio-studio/distro/docker-compose
+```
+
+## Setting up the correct realm configuration in Keycloak
+
+Once your inside the docker-compose workspace. Make the following edits:
+
+- Replace the Keycloak realm configuration with the correct one from Microcks repository. Start by changing to the config directory.
+
+
+```
+cd config/keycloak
+```
+
+- Download the copy of the correct Keycloak realm file from Microcks repository.
+
+
+```
+wget https://raw.githubusercontent.com/microcks/microcks/master/install/keycloak-microcks-realm-full.json
+```
+
+- Rename the existing Microcks realm file to something different.
+
+
+```
+mv microcks-realm.json microcks-realm.json.bkup
+```
+
+- Rename the realm file you downloaded above to `microcks-realm.json`.
+
+
+```
+mv keycloak-microcks-realm-full.json microcks-realm.json
+```
+
+These steps will ensure that you have the correct realm configuration to start the installation. Once done, follow the remaining instructions as-is in the APICurio docker-compose readme here https://github.com/Apicurio/apicurio-studio/blob/master/distro/docker-compose/Readme.md
+
+## Ensuring the "user" role is present correctly in the microcks-app client
+
+Once your installation is up and running, login to Keycloak with your admin credential and follow the following steps:
+
+* Click on Client->*microcks-app*.
+
+* Click on Roles tab and check to confirm if the "*user*" role set correctly. If the role is not present, just create one using the "Add Role" button and give the name of the role as "*user*" and press "Save".
+
+![keycloak-microcks-app-user-role](/images/blog/keycloak-microcks-app-user-role.png)
+
+* Now, click on Client again and move to *microcks-app-js*.
+
+* Here click on "Scope" and ensure *Full Scope allowed* is "ON"
+
+![keycloak-microcks-app-js-scope](images/blog/keycloak-microcks-app-js-scope.png)
+
+
+## Creating users in Microcks Keycloak realm
+
+Now go ahead and create a user in Microcks realm. Once a user is created, Follow the steps below to ensure that the user is setup correctly.
+
+* Ensure that the user has a role called "user" in the client role section under microcks-app client. For checking this, click on the user in the Users page and navigate to the Role Mappings tab. 
+* Here in the Client roles drop-down select *microcks-app.* You should see the an entry called "*user*" in Assigned Roles and Effective Roles sections (both should have it)
+* Once this confirmed. Navigate to Clients->*microcks-app-js*
+* Here, click on the Client scopes and then Evaluate
+* In the user input, enter the name of the user whom you want to check and then click "Evaluate"
+
+![keycloak-client-scope-evaluation.png](/home/narioinc89/Downloads/images/blog/keycloak-client-scope-evaluation.png.png)

```

Comment:
Replace with `![keycloak-client-scope-evaluation.png](/images/blog/keycloak-client-scope-evaluation.png)`

Reply:
This was a silly mistake from my end :) while getting the directory structure I copied in the wrong folder. Changed from my end